core/modules/views/plugins/views_plugin_display_page.inc

```diff
@@ -264,7 +264,7 @@ class views_plugin_display_page extends views_plugin_display {
     $options['path'] = array(
       'category' => 'page',
       'title' => t('Path'),
-      'value' => views_ui_truncate($path, 24),
+      'value' => $path,
```

Comment:
Why is this change here? Was this a mistake?

Reply:
yes, looks like I got a different patch in here.